core/src/main/java/psiprobe/beans/LogResolverBean.java

```diff
@@ -325,11 +325,10 @@ private void interrogateClassLoader(ClassLoader cl, Application application,
       Jdk14ManagerAccessor jdk14accessor = new Jdk14ManagerAccessor(cl);
       jdk14accessor.setApplication(application);
       appenders.addAll(jdk14accessor.getHandlers());
-    } catch (Throwable t) {
+    } catch (ThreadDeath t) {
       // make sure we always re-throw ThreadDeath
-      if (t instanceof ThreadDeath) {
-        throw (ThreadDeath) t;
-      }
+      throw (ThreadDeath) t;
```

Comment:
Sure I will do that. Thank you.

On Wed, Feb 3, 2016 at 8:32 AM, Jeremy Landis <notifications@github.com>
wrote:

> In core/src/main/java/psiprobe/beans/LogResolverBean.java
> <https://github.com/psi-probe/psi-probe/pull/638#discussion_r51677643>:
>
> >        // make sure we always re-throw ThreadDeath
> > -      if (t instanceof ThreadDeath) {
> > -        throw (ThreadDeath) t;
> > -      }
> > +      throw (ThreadDeath) t;
>
> @georgekankava <https://github.com/georgekankava> This is the casting I
> was mentioning. As you have this catching threadDeath now, the casting is
> unnecessary. Can you fix this and squash the commit so I can merge it?
> There are a number of these so please review the overall change. Thanks.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/psi-probe/psi-probe/pull/638/files#r51677643>.
>